### ISSUE TITLE
Project name and version not included in HTML output

### ISSUE THREAD

**Issue by Abbe98** [documentationjs/gulp-documentation#35]
Currently project version & name is not included in the generated HTML title as expected:

![image](https://cloud.githubusercontent.com/assets/2631719/25625734/95e987d8-2f5d-11e7-884b-6d33b41ff450.png)

Nor when you define them in the task:

````js
gulp.task('docs', function () {
  return gulp.src('./src/index.js')
    .pipe(gulpDocumentation('html'), {}, {
      name: 'Project.js'
    })
    .pipe(gulp.dest('docs'));
});
```

**Comment by rodrigogrow** [comment]
I have the same problem here..

**Comment by anthony-redFox** [comment]
Fixed in documentationjs/documentation#845 and released v5.1.1
